src/ert/scheduler/scheduler.py

```diff
@@ -95,8 +96,8 @@ def __init__(
             real.iens: Job(self, real) for real in (realizations or [])
         }
 
+        self._loop = get_running_loop()
```

Comment:
I'm still wondering whether `get_running_loop` should just crash when there is no loop running - instead of creating one. As this is the basic behavior of asyncio https://docs.python.org/3/library/asyncio-eventloop.html opposite to our version.

Reply:
Yes, but why would we crash instead of handling it by creating a new one?

Reply:
Depends what we expect. Do we expect that there is already one? Maybe we don't care :smile: In that case it is fine. Let's just say it's fine then.